### DISCUSSION TITLE
On my M1 iPad, windows runs horribly

### DISCUSSION THREAD

**Discussion by vervamond** [utmapp/UTM#4495] [UTM Help (iOS)]
Anything I could do to optimize it?

**Configuration (required)**
* UTM Version: 4.0.6
* OS Version: 15.5 b4
* Device Model: iPad Air 5
* Is it jailbroken (name jailbreak used)? TrollStore 
* How did you install UTM? TrollStore 

N/A

**Upload VM**
I just lost my VM by accident. I’ll reinstall it soon.

**Comment by osy** [comment]
There’s nothing you can do. There’s no windows GPU acceleration which is why it feels sluggish.

**Reply by vervamond** [reply]
Weird. It seems to run much worse than in the video (https://youtu.be/lDZOjIGHka0)

**Reply by osy** [reply]
Can you provide a video?

**Reply by vervamond** [reply]
I did: https://youtu.be/lDZOjIGHka0

**Reply by osy** [reply]
That is my video not yours…

**Reply by vervamond** [reply]
Oh sorry

**Reply by vervamond** [reply]
I don’t have my vm working run. I might be able to do it tomorrow.
